microshift_release_notes/microshift-4-13-release-notes.adoc

```diff
@@ -186,4 +186,13 @@ Issued: 2023-06-13
 
 * Previously, when the operating system was not set to the default route, OVN-Kubernetes caused the cluster to function improperly. With this update, the default routes were removed and the OVN-Kubernetes pod runs as expected. (link:https://issues.redhat.com/browse/OCPBUGS-13548)([*OCPBUGS-13548*])
 
+For the `TopoLVM image`, see link:https://catalog.redhat.com/software/containers/lvms4/topolvm-rhel9/63972de3adcb55263891b983?container-tabs=dockerfile[lvms4/topolvm-rhel9].
```

Comment:
I see this reference a bit earlier too at line 175. Should it be duplicated?

Reply:
Yeah for some reason git tracks the changes on line 189 as the new changes for this entry and the ones on like 198 as the old ones even though it is the other way around. It does render correctly in the preview. 🤷‍♀️